src/main/java/resources/PersonResource.java

```diff
@@ -115,4 +115,15 @@ public Person updatePerson(Person person) {
         personDao.update(person);
         return person;
     }
+
+    @PUT
+    @Path("/{id}/switchAccountStatus")
+    @UnitOfWork
+    public Response switchAccountStatus(@PathParam("id") Long id) {
```

Comment:
You're right, my bad. Still, it's better to have 2 endpoints then, one which activates and one which deactivates.